src/main/java/org/zalando/nakadi/service/EventStream.java

```diff
@@ -122,7 +122,8 @@ public void streamEvents(final AtomicBoolean connectionReady) {
         } catch (final IllegalStateException e) {
             LOG.info("Error occurred when streaming events (possibly server closed connection)", e);
         } catch (final KafkaException e) {
-            LOG.error("Error occurred when polling events from kafka", e);
+            LOG.error("Error occurred when polling events from kafka; consumer: " + config.getConsumingAppId() +
```

Comment:
you can pass it as the latest param
https://www.slf4j.org/faq.html#paramException